app/controllers/external_link_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ExternalLinkController < ApplicationController
+  layout nil
+
+  def show
+    person = Person.find(params[:person_id])
+    link_name = params[:link_name]
+    url = person.send("#{link_name}_url")
```

Comment:
Run `brakeman` on this please. I feel like we may have a slight security gap here.